config/config.go

```diff
@@ -308,6 +308,7 @@ type OpenIdConfig struct {
 	IssuerUri               string            `yaml:"issuer_uri,omitempty"`
 	Scopes                  []string          `yaml:"scopes,omitempty"`
 	UsernameClaim           string            `yaml:"username_claim,omitempty"`
+	AllowedDomains          []string          `yaml:"allowed_domains,omitempty"`
```

Comment:
Also, please keep the alphabetical order :-)

Reply:
ack